src/main/java/com/codeit/sb01_deokhugam/domain/review/entity/Review.java

```diff
@@ -0,0 +1,84 @@
+package com.codeit.sb01_deokhugam.domain.review.entity;
+
+import java.math.BigDecimal;
+
+import com.codeit.sb01_deokhugam.domain.base.BaseUpdatableEntity;
+import com.codeit.sb01_deokhugam.domain.book.entity.Book;
+import com.codeit.sb01_deokhugam.domain.user.entity.User;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.Table;
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@Getter
+@Entity
+@Table(name = "review")
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+public class Review extends BaseUpdatableEntity {
+
+	@ManyToOne(fetch = FetchType.LAZY, optional = false)
+	@JoinColumn(name = "user_id", columnDefinition = "uuid NOT NULL")
+	private User user;
```

Comment:
큰 부분은 아닌데 User 타입의 필드변수가`user` -> `author` 같은 의미를 나타내는 변수명은 어떤가요?

```suggestion
	@ManyToOne(fetch = FetchType.LAZY, optional = false)
	@JoinColumn(name = "user_id", columnDefinition = "uuid NOT NULL")
	private User author;
```

어떻게 생각하시는지 알려주세요 😄

Reply:
author로 변수명을 바꾸는게 더 의미있는 변수명이 될 듯 합니다. 다음 커밋에 반영시켜두겠습니다!